etl/steps/data/garden/ggdc/2020-10-01/ggdc_maddison.meta.yml

```diff
@@ -5,14 +5,14 @@ tables:
         title: Population
         unit: people
         display:
-          entityAnnotationsMap: "Western Offshoots (MPD): United States, Canada, Australia and New Zealand"
+          entityAnnotationsMap: "Western offshoots (MPD): United States, Canada, Australia and New Zealand"
```

Comment:
Keep in mind that changing this file will trigger changes in many etl steps. So, it would be good to merge at the end of the day, to avoid blocking others. Also, there will be many charts to review.